apps/jenkins/deployers/jenkins.yml

```diff
@@ -1,41 +1,47 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: jenkins
+  labels:
+    lkey: lvalue
+    affinity_label: jenkins
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        lkey: lvalue
+        affinity_label: jenkins
+    spec:
+      securityContext:
+         fsGroup: 1000
+      containers:
+      - name: jenkins
+        imagePullPolicy: IfNotPresent
+        image: jenkins/jenkins:lts
+        ports:
+        - containerPort: 8080
+        volumeMounts:  
+        - mountPath: /var/jenkins_home
+          name: jenkins-home 
+      volumes: 
+      - name: jenkins-home
+        persistentVolumeClaim: 
+          claimName: testclaim
+---
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: testclaim
+  labels:
+    affinity_label: jenkins
```

Comment:
lets take it as lvalue

Reply:
OK done. Relaced and tetsted. working fine.
```
[root@master-1554817485 deployers]# kubectl get pods -n app-jenkins-ns -o wide
NAME                      READY     STATUS    RESTARTS   AGE       IP            NODE
jenkins-8f9bbd4b7-s86zg   1/1       Running   0          5m        172.23.6.76   node3-1554817485.mayalabs.io
[root@master-1554817485 deployers]# kubectl get pvc -n app-jenkins-ns
NAME            STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS         AGE
jenkins-claim   Bound     pvc-367df512-86aa-11e9-9a67-00505698550d   5G         RWO            openebs-cstor-disk   5m
[root@master-1554817485 deployers]# kubectl get pods -n openebs -o wide | grep pvc-367d
pvc-367df512-86aa-11e9-9a67-00505698550d-target-7ddb9697d-9q996   3/3       Running   1          5m        172.23.6.75    node3-1554817485.mayalabs.io
```